requirements/base.txt

```diff
@@ -31,7 +31,7 @@ pyyaml==5.4.1
 sqlalchemy==1.3.17
 pymysql==0.9.3
 requests==2.22.0
-elasticsearch==7.13.4
+elasticsearch==7.16.2
```

Comment:
can you please keep the version to be 7.13.4 since aws es7 no longer works after that version? context: https://github.com/pinterest/querybook/pull/685#issuecomment-975894514